scipy/optimize/tests/test_chandrupatla.py

```diff
@@ -560,7 +560,7 @@ def f(x):
                               reason='JAX arrays do not support item assignment.')
 @pytest.mark.skip_xp_backends('cupy',
                               reason='cupy/cupy#8391')
-class TestChandrupatla(TestScalarRootFinders):
+class TestFindRoot(TestScalarRootFinders):
```

Comment:
As far as I could tell this inheritance doesn't do anything?

Reply:
Maybe it gives access to some functions that are attributes? It's not part of the PR, so I don't know/remember.

Reply:
I tested locally and it's unused. I am happy to push a commit removing it but permission was denied

Reply:
Weird. "Maintainers are allowed to edit this pull request."

Reply:
```suggestion
class TestFindRoot:
```

Reply:
Mm better do this locally and remove the import

Reply:
Done.